bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -21,6 +21,7 @@ export class LegendView extends AnnotationView {
   protected text_widths: Map<string, number>
   protected title_height: number
   protected title_width: number
+  protected max_glyph_size: number
```

Comment:
no need to store this on the view I don't think. If you make the method a property getter you can do
```
const {max_gylph_size} = this.model
```
wherever the value is needed.

Reply:
I modeled this after `const legend_names = this.model.get_legend_names()`  (~ ln 40); would you want that to change also?

Reply:
I suppose this is for some level of caching results. That seems plausible for the text dimensions since competing those can be expensive, but I think the `max_glyph_size` is probably not expensive enough to need caching. Will defer to @mattpap though

Reply:
Moved max_glyph_size out of the view.